artview/plugins/__init__.py

```diff
@@ -28,7 +28,7 @@
         tmp = importlib.import_module('.'+module[:-3], __package__)
     for plugin in tmp._plugins:
         setattr(thismodule, plugin.__name__, plugin)
-        _plugins.append(plugin)
+        _plugins[plugin.__name__] = plugin
```

Comment:
here none, in modes however I can do

    for key in _plugins.keys():
        name = key

instead of

    for plugin in _plugins:
        name = plugin._name_

yes, the diference is minimal!